detection-rules/impersonation_employee_payroll_fraud.yml

```diff
@@ -19,10 +19,10 @@ source: |
   )
   and 1 of (
     regex.icontains(body.current_thread.text,
-                    '(pay\s?(roll|check|date|day)|direct deposit|\bACH\b|\bdd\b|gehalt|salario|salary)'
+                    '(pay\s?(roll|check|date|day)|direct deposit|paid.{0,50}problems|\bACH\b|\bdd\b|gehalt|salario|salary)'
     ),
     regex.icontains(subject.subject,
-                    '(pay\s?(roll|check|date|day)|direct deposit|\bACH\b|\bdd\b|gehalt|salario|salary)'
+                    '(pay\s?(roll|check|date|day)|direct deposit|(acct|account) rephrase|\bACH\b|\bdd\b|gehalt|salario|salary)'
```

Comment:
should we add both new keywords to both subject.subject and body.current_thread.text?

Reply:
Ah yeah, will fix, thanks!